src/main/java/lotto/controller/LottoController.java

```diff
@@ -10,24 +11,31 @@
 public class LottoController {
 
   public void lotto() {
-    generateLotto();
-  }
-
-  private void generateLotto() {
-    LottoList lottoList = payToCounter();
+    Counter counter = payToCounterTemp();
+    OutputView.printCompletePurchaseLotto(counter);
     WinningLotto winningLotto = winningLottoNumber();
-    generateLottoResult(lottoList, winningLotto);
+    generateLottoResult(counter.getLottoList(), winningLotto);
   }
 
-  private LottoList payToCounter() {
-    Counter counter = new Counter(InputView.inputMoneyPurchaseLotto());
-    LottoList lottoList = counter.getLottoList();
-    OutputView.printCompletePurchaseLotto(lottoList);
-    return lottoList;
+  private Counter payToCounterTemp() {
+    try {
+      Counter counter = new Counter(InputView.inputMoneyPurchaseLotto());
+      counter.issueLotto(InputView.inputManualPurchaseLotto());
```

Comment:
```suggestion
      Counter counter = new Counter(InputView.inputMoneyPurchaseLotto(), InputView.inputManualPurchaseLotto());
```
생성자를 통해 생성하고 `issueLotto`를 호출해야만 유효한 `Counter`가 생성되는거라면
한번에 생성하도록 하는건 어떨까요?🤔